### ISSUE TITLE
Move beyond version PMP 2.34

### ISSUE THREAD

**Issue by flying-sausages** [knapsu/plex-media-player-flatpak#3]
https://github.com/knapsu/plex-media-player-flatpak/blob/master/tv.plex.PlexMediaPlayer.json#L42

We are now on version 2.58 with the media player, it might be nice to update this. Is there maybe a way to dynamically update this or automate it?

There are ways to use the github api to retrieve the last version and source file zip/tar. SHAs aren't part of it, but that could easily be done in the same script that would update the json.

**Comment by knapsu** [comment]
Using GitHub API to get info about releases is not a problem. The reasons automation was not yet done because of Freedesktop Platform image that has `cmake` without SSL support. This made it impossible to download anything using HTTPS. Latest images has this fixed but those can not be used because it contains too new Qt libraries which are not recommended by Plex to be used.

After reading your issue I decided to make a workaround by compiling cmake as a prebuild step for building Plex Media Player. It took me quite some time but I managed to do it.

Soon the Flatpak repository I host on my website will contain the latest packages.

Doing final tests that will automate building and publishing Flatpak image when a new official release will be detected.
